config/webpack.test.js

```diff
@@ -1,76 +1,32 @@
 /* eslint strict: 0 */
 'use strict';
 
-const common = require('./webpack.common');
 const path = require('path');
+const common = require('./webpack.common');
+const externals = {
+  jquery: 'jQuery',
+};
 
-module.exports = {
-
-  mode: 'production',
-
-  optimization: {
-    minimize: false
-  },
-
-  devtool: 'source-map',
-
-  entry: './test/test.js',
-
-  output: {
-    path: path.resolve('./build/test'),
-    pathinfo: true,
-    filename: 'test.bundle.js'
-  },
-
-  resolve: {
-    modules: ["node_modules"],
-    alias: {
-      'emailjs-stringencoding': path.resolve('./src/lib/string-encoding'),
-      'text-encoding': path.resolve('./src/lib/string-encoding')
```

Comment:
These two lines have been added in https://github.com/mailvelope/mailvelope/commit/8a4495ad1c99fa4a235068dec46cef165f441dfb and should not be deleted.